db/database.go

```diff
@@ -79,6 +80,7 @@ type DatabaseContext struct {
 	State              uint32                  // The runtime state of the DB from a service perspective
 	ExitChanges        chan struct{}           // Active _changes feeds on the DB will close when this channel is closed
 	OIDCProviders      auth.OIDCProviderMap    // OIDC clients
+	PurgeInterval      int                     // Metadata purge interval, in hours
```

Comment:
I considered adding it to the bucket, but left it here to avoid casting the bucket to GoCouchbaseBucket every time we needed the value.   Although it's only strictly needed for Couchbase at this point (because walrus doesn't support xattrs), I don't know if it really merits a 'Couchbase' prefix.